### PR TITLE
DROOLS-5766 : Issue while using LocalDateTime field reference in GRE

### PR DESCRIPTION
**Pull request by Rikkola**
**JIRA**: 

[link](https://issues.redhat.com/browse/DROOLS-5766)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelPersistenceHelper.java [hunk 1]
```diff
@@ -537,8 +532,6 @@ static String adjustParam(final String dataType,
                 return param.substring(0,
                                        param.length() - 1);
             }
-        } else if (boundParams.containsKey(param)) {
-            return "=" + param;
```
Comment (Rikkola) [inline drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelPersistenceHelper.java:541]: The fix was to remove this bound parameters check. I could not find a single unit test hitting this spot, but I know this was added before the GRE moved to DRL based save format. The "=" here is not needed.

**Comment by jstastny-cz** [comment]
jenkins do fdb

**Comment by jstastny-cz** [comment]
StackOverflow issue during in FDB: `[INFO] Business Central - Webapp .......................... FAILURE [24:17 min]` 
```
ERROR] Failed to execute goal io.thorntail:thorntail-maven-plugin:2.7.0.Final:package (default) on project business-central-webapp: Unable to create thorntail.jar: Failed to write asset to output: /_bootstrap/business-central-webapp.war: StackOverflowError -> [Help 1]
[2020-11-03T17:20:35.569Z] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.thorntail:thorntail-maven-plugin:2.7.0.Final:package (default) on project business-central-webapp: Unable to create thorntail.jar
```

**Comment by Rikkola** [comment]
Should not be related.
jenkins do fdb

**Comment by jstastny-cz** [comment]
jenkins do fdb

**Comment by jstastny-cz** [comment]
jenkins do fdb

**Comment by Rikkola** [comment]
> org.jbpm.springboot.samples.CustomNotificationListenerTest.testProcessEventListenerRegistration
> org.jbpm.springboot.samples.CustomNotificationListenerTest.testProcessEventListenerRegistration

jenkins do fdb

**Comment by karreiro** [comment]
Jenkins do fdb.

**Comment by Rikkola** [comment]
@karreiro @jstastny-cz Finally green.

**Review by jstastny-cz — APPROVED** [review]
LGTM, tested with the enclosed reproducer.

**Review by karreiro — APPROVED** [review]
Thank you, @Rikkola! Great investigation :-)

**Comment by Rikkola** [comment]
@kiegroup/gatekeepers Hello, can I get a merge please.

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**